pkgs/native_synchronization/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Files and directories created by pub.
```

Comment:
Are some of these covered by the top-level `.gitignore`?

Reply:
We don't have a top-level gitignore here. And general, I think it makes sense for each package to be self-contained at least from the POV of having a complete gitignore file.